extensions/amp-story-interactive/validator-amp-story-interactive.protoascii

```diff
@@ -86,133 +92,192 @@ tags: {  # <amp-story-interactive-results>
+    name: "option-4-results-threshold"
+    value_regex: "\d+[.\d+]?"
+    trigger: {
+      also_requires_attr: "option-4-results-category"
+    }
   }
   attrs: {
-    name: 'option-4-text'
+    name: "prompt-text"
   }
   attrs: {
-    name: 'prompt-text'
+    name: "theme"
+    value: "light"
+    value: "dark"
   }
   attrs: {
-    name: 'theme'
-    value: 'light'
-    value: 'dark'
+    name: "chip-style"
+    value: "flat"
+    value: "transparent"
   }
 }
 
 attr_lists: {
-  name: 'interactive-shared-configs-attrs'
+  name: "interactive-shared-configs-attrs"
   attrs: {
-    name: 'id'
+    name: "id"
     mandatory: true
   }
   attrs: {
-    name: 'prompt-text'
+    name: "prompt-text"
   }
   attrs: {
-    name: 'endpoint'
+    name: "endpoint"
     mandatory: true
     value_url: {
-      protocol: 'https'
+      protocol: "https"
       allow_relative: false
       allow_empty: false
     }
   }
   attrs: {
-    name: 'theme'
-    value: 'light'
-    value: 'dark'
+    name: "theme"
+    value: "light"
+    value: "dark"
   }
   attrs: {
-    name: 'chip-style'
-    value: 'shadow'
-    value: 'flat'
-    value: 'transparent'
+    name: "chip-style"
+    value: "shadow"
+    value: "flat"
+    value: "transparent"
   }
   attrs: {
-    name: 'prompt-size'
-    value: 'small'
-    value: 'medium'
-    value: 'large'
+    name: "prompt-size"
+    value: "small"
+    value: "medium"
+    value: "large"
   }
 }
 attr_lists: {
-  name: 'interactive-options-text-attrs'
+  name: "interactive-options-text-attrs"
   attrs: {
-    name: 'option-1-text'
+    name: "option-1-text"
     mandatory: true
   }
   attrs: {
-    name: 'option-2-text'
+    name: "option-2-text"
     mandatory: true
   }
   attrs: {
-    name: 'option-3-text'
+    name: "option-3-text"
   }
   attrs: {
-    name: 'option-4-text'
+    name: "option-4-text"
+    trigger: {
+      also_requires_attr: "option-3-text"
+    }
   }
 }
 attr_lists: {
-  name: 'interactive-options-confetti-attrs'
+  name: "interactive-options-confetti-attrs"
   attrs: {
-    name: 'option-1-confetti'
+    name: "option-1-confetti"
   }
   attrs: {
-    name: 'option-2-confetti'
+    name: "option-2-confetti"
   }
   attrs: {
-    name: 'option-3-confetti'
+    name: "option-3-confetti"
   }
   attrs: {
-    name: 'option-4-confetti'
+    name: "option-4-confetti"
   }
 }
 attr_lists: {
-  name: 'interactive-options-results-category-attrs'
+  name: "interactive-options-results-category-attrs"
   attrs: {
-    name: 'option-1-results-category'
+    name: "option-1-results-category"
+    trigger: {
+      also_requires_attr: "option-2-results-category"
+    }
   }
   attrs: {
-    name: 'option-2-results-category'
+    name: "option-2-results-category"
+    trigger: {
+      also_requires_attr: "option-1-results-category"
+    }
   }
   attrs: {
-    name: 'option-3-results-category'
+    name: "option-3-results-category"
+    trigger: {
+      also_requires_attr: "option-1-results-category"
```

Comment:
Did you mean 2?

Reply:
It really is the same because option1 requires option2 and viceversa. I see that setting option-2 here makes a bit more sense when you read it, so I'll change it